tools-local/init-build.proj

```diff
@@ -105,12 +105,10 @@
   <Target Name="ExtractToolPackage"
           DependsOnTargets="BuildTasks"
           Condition="'$(OfflineBuild)' == 'true'">
-          <Message Importance="High" Text="Arcversion ->> $(ArcadeSdkOverride.Version)    SBPackPath ->> $(SourceBuiltPackagesPath)" />
     <ItemGroup>
       <_ToolPackage
-        Condition="'%(ArcadeSdkOverride.Version)' != ''"
-        Include="$(SourceBuiltPackagesPath)%(ArcadeSdkOverride.Identity).%(ArcadeSdkOverride.Version).nupkg"
-        Exclude="$(SourceBuiltPackagesPath)%(ArcadeSdkOverride.Identity).%(ArcadeSdkOverride.Version).symbols.nupkg"
+        Include="$(SourceBuiltPackagesPath)%.1.0.0-dev.nupkg"
```

Comment:
A couple of comments:
1. SourceBuildPackagesPath should be PrebuiltSourceBuiltPackagesPath, since this is a prebuilt from the previous source-build build.
2. You need the name of the package, and no "%".  Microsoft.DotNet.Arcade.Sdk.1.0.0-dev.nupkg

Reply:
Also, once you get the tool package extracted to the packages dir, you'll need to update the SOURCE_BUILT_SDK  EnvironmentVariables in arcade.proj to not be conditional and you can also remove the "alternate-workaround-for-bootstrapping" patch from patches/arcade.